index.html

```diff
@@ -1,544 +1,627 @@
-                <div class="image">
-                  <img src="./img/tab-1.jpg" class="modal-button">
-                </div>
-              </div>
-              <div class="card-content">
-                <div class="content">
-                  <div>Best customer's story</div>
-                  <p>We have achieved customer satisfaction and income increase after having implemented accessibility.</p>
-                  <p>That's a real magic!</p>
-                  <span class="button is-link modal-button" data-target="modal-image1">Learn more</span>
-                </div>
-              </div>
-            </div>
-          </div>
-          <div class="column is-6">
-            <div class="card">
-              <div class="card-image">
-                <div class="image">
-                  <img src="./img/tab-2.jpg">
-                </div>
-              </div>
-              <div class="card-content">
-                <div class="content">
-                  <div>Other Best customer's story.</div>
-                  <p>We were not so sure in success but we have had lots of new customers after implementing accessibility.</p>
-                  <p>We will recommend Best Company to our friends!</p>
-                  <span class="button is-link modal-button" data-target="modal-card">Learn more</span>
-                </div>
-              </div>
-            </div>
+      <div class="hero-bg hero-body is-fullheight">
+        <div class="container has-text-centered">
+          <div class="column is-10 is-offset-1">
+            <h1 class="title has-text-white">We have web accessibility in
+              mind</h1>
           </div>
         </div>
       </div>
-    </div>
 
-    <div class="tab-pane" id="pane-3">
-      <div class="columns medias">
-        <div class="column">
-          <div class="media">
-            <div class="media-left">
-              <i class="fab fa-empire fa-4x"></i>
-            </div>
-            <div class="media-content">
-              <div class="content">
-                <p>
-                  <a><b>Solution architect</b></a>
-                  <br> Please read about our team happiness
+      <nav class="breadcrumb container" aria-label="breadcrumbs">
+        <ul>
+          <li><a href="#">Home</a></li>
+          <li class="is-active" aria-current="page"><a href="#">Who we are?</a></li>
+        </ul>
+      </nav>
+
+      <section class="section" id="subscription">
+        <div class="section-heading has-text-centered">
+          <div class="column is-8 is-offset-2">
+            <h2 class="title is-2">Would you like to get the latest news?</h2>
+            <h3 class="subtitle is-5">Subscribe to our events, updates and
+              more</h3>
+            <form class="box has-background-grey-light">
+              <div class="field is-grouped">
+                <p class="control is-expanded">
+                  <input type="email" class="input" placeholder="Your email">
```

Comment:
As the input doesn't have corresponding label and it won't be clear for the user with screen reader what the field is about, I would suggest to add a label with class visually-hidden. Thus screen reader will be able to read this content

Reply:
I understand this, but wouldn't `type="email"` is not clear enough?

Reply:
It is considered to be a good practice to have an input associated with the label. Otherwise it violates [Success Criterion 3.3.2: Labels or Instructions (Level A)](https://wcag.com/developers/3-3-2-labels-instructions/)